lib/RoadizCoreBundle/src/Entity/NodeType.php

```diff
@@ -433,7 +433,7 @@ public function getSourceEntityTableName(): string
 
     public function __toString(): string
     {
-        return (string) $this->getId();
+        return (string) $this->getName();
```

Comment:
```suggestion
        return $this->getName();
```

getName already ensure string return type